e2e/tables.spec.ts

```diff
@@ -276,10 +276,16 @@ test.describe('Tables', () => {
         })
 
       await test.step('add new columns', async () => {
+        // wait for previously entered column schemas to appear
+        // ...so addColumnsFromColumnSchemaConfig calculates correct starting column index
+        await expect(
+          userPage.getByPlaceholder('Column name'),
+          'should load previously entered column schemas',
+        ).toHaveCount(initialColumnsSchemaConfig.length)
```

Comment:
In webkit in CI, the table column schema editor loaded inconsistently, so the starting column index would intermittently be calculated before the previous column schemas had loaded and result in test failures.